packages/libs/react-components/src/components/Button/Button.stories.tsx

```diff
@@ -19,6 +21,12 @@ const meta: Meta<
         type: 'select',
       },
     },
+    selectVariant: {
+      options: ['primaryFilled', 'secondaryFilled', 'positiveFilled'],
```

Comment:
nit: You can take the keys from the variant object so that they don't get out of sync when adding/removing variants